src/transform.rs

```diff
@@ -16,9 +16,10 @@ fn escape_html(text: impl AsRef<str>) -> Result<String, Error> {
     Ok(out)
 }
 
-/// TODO: This should likely not require a db connection, the caller should provide a map of usernames to
```

Comment:
Note that the initial suggestion of a hashmap was flawed - we have a _very_ large number of candidate accounts to look through, and we want to use the database for this. For that reason, this patch just moves the lookup out of the `bio()` function into a passed function.

Reply:
That is perfect - the behavior that I thought would eventually be the aforementioned map would just be a list of viable candidates "known" by the caller, and this would read-through on miss to the db. Moving it into a function nicely defers that sort of ugliness to someone willing to bear it tho. Good call.